mutornadomon/monitor.py

```diff
@@ -70,6 +107,12 @@ def __init__(self, host_limit=r'.*', request_filter=LOCALHOST, io_loop=None):
         self._GAUGES = {}
         self._reset_ephemeral()
 
+    def _publish(self):
+        try:
+            self.publisher(self.metrics)
+        except:
```

Comment:
This was actually a bit of (too much?) defensive programming. I treat this callback as user input and in theory it can raise something other than `Exception` that inherits from `BaseException`.